angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -1,9 +1,19 @@
 <md-card class="widget-frame" id="portlet-id-{{portlet.nodeId}}" aria-label="{{ portlet.title }} widget">
 
+	<!-- MAINTENANCE MODE OVERLAY -->
+	<div class="overlay__maintenance-mode" ng-if="portlet.widgetConfig.maintenanceMode">
+		<div class="maintenance-content">
+			<p>
+				<md-icon class="md-warn">warning</md-icon>
+			</p>
+			<p>This app is undergoing scheduled maintenance and is temporarily unavailable.</p>
```

Comment:
I'd drop the "scheduled" so we're more free to use this maintenance mode in a wider variety of situations.